pkg/lvm/lvm_util.go

```diff
@@ -49,6 +49,7 @@ const (
 	VGCreate = "vgcreate"
 	VGList   = "vgs"
 
+	LVList   = "lvs"
```

Comment:
LVS command is already added [here](https://github.com/openebs/lvm-localpv/pull/94/files#diff-1d348d11d677906b0fb0a0506d50f2bfed43754cc0f16b6509a3bef047095722R56). You can remove it and keep the `LVList`